sklearn/utils/tests/test_random.py

```diff
@@ -180,3 +184,28 @@ def test_random_choice_csc_errors():
     class_probabilities = [np.array([0.5, 0.6]), np.array([0.6, 0.1, 0.3])]
     assert_raises(ValueError, random_choice_csc, 4, classes,
                   class_probabilities, 1)
+
+
+@pytest.mark.parametrize("low,high,base", [
+    (-4, 0, 10),
+    (-6, 1, np.exp(1)),
+    (-8, 4, 2),
+])
+def test_log_uniform(low, high, base):
+    rv = loguniform(low, high, base=base)
+    rvs = rv.rvs(size=100)
+    assert (base**low <= rvs).all() and (rvs <= base**high).all()
+
+    y = loguniform(low, high)
+    rvs = y.rvs(size=100)
```

Comment:
I'm not sure this is necessary. I know it's just to check the default base. If we need that, just test the defaults in a separate test.